MsGraphicsPkg/SimpleWindowManagerDxe/WaitForEvent.c

```diff
@@ -153,14 +153,21 @@ WaitForEventInternal (
           PowerTitle   = HiiGetString (mSWMHiiHandle, STRING_TOKEN (STR_POWER_TIMEOUT_TITLE), NULL);
           PowerBody    = HiiGetString (mSWMHiiHandle, STRING_TOKEN (STR_POWER_TIMEOUT_BODY), NULL);
           PowerCaption = HiiGetString (mSWMHiiHandle, STRING_TOKEN (STR_POWER_TIMEOUT_CAPTION), NULL);
-          Status       = SwmDialogsMessageBox (
-                           PowerTitle,                               // Dialog body title.
-                           PowerBody,                                // Dialog body text.
-                           PowerCaption,                             // Dialog caption text.
-                           SWM_MB_CANCEL | SWM_MB_STYLE_ALERT2,      // Show Cancel and enforce Timeout
-                           EFI_TIMER_PERIOD_SECONDS (PcdGet16 (PcdPowerOffHold)),
-                           &SwmResult
-                           );                                         // Return result.
+          // MU_CHANGE [BEGIN] - CodeQL change
+          if ((PowerTitle == NULL) || (PowerBody == NULL) || (PowerCaption == NULL)) {
+            DEBUG ((DEBUG_ERROR, "Error getting POWER OFF dialog strings. Code = %r\n", Status));

```

Comment:
Shouldn't you be updating the status here?  It should always be EFI_SUCCESS at this point.

Reply:
Updated